osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -14,53 +15,92 @@ namespace osu.Game.Rulesets.Osu.Difficulty.Skills
     /// </summary>
     public class Aim : OsuStrainSkill
     {
-        private const double angle_bonus_begin = Math.PI / 3;
-        private const double timing_threshold = 107;
-
         public Aim(Mod[] mods)
             : base(mods)
         {
         }
 
+        protected override int HistoryLength => 2;
+
+        private const double wide_angle_multiplier = 1.5;
+        private const double acute_angle_multiplier = 2.0;
+
         private double currentStrain = 1;
 
-        private double skillMultiplier => 26.25;
+        private double skillMultiplier => 23.25;
         private double strainDecayBase => 0.15;
 
         private double strainValueOf(DifficultyHitObject current)
         {
-            if (current.BaseObject is Spinner)
+            if (current.BaseObject is Spinner || Previous.Count <= 1 || Previous[0].BaseObject is Spinner)
```

Comment:
The check for previous is needed due to how movement time and traveltime are zero when the next object is a spinner. It doesnt matter if previous[1] is a spinner since we dont look at that objs times or distances.